docs/DesignDocs/SGX_QuoteEx_Integration.md

```diff
@@ -139,10 +139,19 @@ function takes an input attestation key ID in its parameter list.
 With DCAP, the quote generation can be done either in-process,
 or out-of-process by working with a background service (called AESM) running
 on the same platform.
+  - For in-process quote generation, the process that calls the DCAP library
+  is required to run in an account added to a special group `sgx_prv`,
+  as documented in the DCAP library [readme](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/README.md)
+  and the DCAP driver [readme](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/driver/linux/README.md).
+    - Note: `sgx_prv` is an SGX provisioning access control mechanism implemented
+    in the SGX Linux driver since version V1.22, and in the
+    upstream SGX Linux driver.
```

Comment:
Please add the equivalent statements about Windows, since the rest of this document does not seem to be specific to Linux

Reply:
Good catch! will add Windows-specific content.

Reply:
Thanks!